src/Cogworks.CogFlare.Core/README.md

```diff
@@ -52,12 +52,30 @@ Add these settings to the **appsettings.json**
     "Endpoint": "https://api.cloudflare.com/client/v4/zones/[zoneId]/purge_cache",
     "KeyNodes": "1234, 031089",
     "IsEnabled": true,
-    "BlockListPropertyAlias": "",
     "BlockAliases": "formBlock,newBlock",
     "Domain": "https://www.example.com"
   }
 ```
 
+For bypass logic you can use 2 options :
+1. Adding a new property "disableCloudFlareCache" in the node where you want to disable cache :
```

Comment:
Remove space before the : please (unless it's some special md character I don't know about)